backend/application/tests/fakes.py

```
@@ -38,25 +38,22 @@ def create(self, question: Question) -> Question:
         self.questions.append(saved)
         return saved  # DB の自動採番をエミュレートするため連番 ID を割り当てて返す
 
-    def update(self, question_id: int, updates: dict) -> Question | None:
+    def update(self, question_id: QuestionId, patch: QuestionPatch) -> Question | None:
         for i, q in enumerate(self.questions):
-            if q.id == question_id:
-                english_raw = updates.get("english")
-                japanese_raw = updates.get("japanese")
-                tags_raw = updates.get("tags")
+            if q.id == question_id:  # QuestionId 同士の比較（dataclass equality）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)